packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemFile.kt

```diff
@@ -33,7 +49,13 @@ class FileSystemFile(file: File) : FileSystemPath(file) {
     }
   }
 
+  fun asString(): String? {
+    val uriString = Uri.fromFile(path).toString()
+    return if (uriString.endsWith("/")) uriString.dropLast(1) else uriString
+  }
```

Comment:
I'm not sure about this. It's looking weird. I would instead return `uriString` rather than remove the `/`.

Reply:
I want to keep convention that file paths end without a slash (so usually `/file.txt`) and folders end with a slash (`/path/to/folder/`)